internal/literals/literals.go

```diff
@@ -179,6 +219,12 @@ func Obfuscate(files []*ast.File, info *types.Info, fset *token.FileSet, blackli
 	return files
 }
 
+//go:linkname isInteger go/types.isInteger
+func isInteger(typ types.Type) bool
```

Comment:
This is already check extensively in obfuscateNumberLiteral, no need to add these functions.